packages/server/src/insights/insight-classes.ts

```diff
@@ -166,11 +162,10 @@ export class AverageWaitTime implements InsightInterface<QuestionModel> {
   displayName = 'Average Wait Time';
   description = 'Gets the average wait time';
   roles = [Role.PROFESSOR];
-  component = InsightDisplay.SimpleDisplay;
+  component = InsightDisplay.SimpleChart;
```

Comment:
Wait is this a chart tho? I thought the way we query this is we get the literal average wait time of all questions (with filters applied), therefore it should be a SimpleDisplay

Reply:
Good catch, I just fixed that.